discovery/sync_manager.go

```diff
@@ -270,7 +270,8 @@ func (m *SyncManager) syncerHandler() {
 			case len(m.activeSyncers) == 0 &&
 				len(m.inactiveSyncers) == 0:
 
-				attemptHistoricalSync = true
+				attemptHistoricalSync =
```

Comment:
Looks like we also need to pause the ticker responsible for scheduling historical syncs after the initial one?

Reply:
if `m.cfg.NumActiveSyncers == 0` there shouldn't be an initial sync at all?

Reply:
Yeah should be good now since the ticker isn't started until the initial historical sync is started.